### ISSUE TITLE
DependencySourcesItem.targets in bsp4j is List, but single item in bsp4s

### ISSUE THREAD

**Issue by jastice** [scalacenter/bsp#37]
In bsp4j:
https://github.com/scalacenter/bsp/blob/99c53ab1752f8f724e35eb461a9f4cbbe4b295db/bsp4j/src/main/java/ch/epfl/scala/bsp4j/Protocol.xtend#L342

In bsp4s:
https://github.com/scalacenter/bsp/blob/eb45419f65c7f23df7e5de7bc109bc517f6b9f88/src/main/scala/ch/epfl/scala/bsp/Bsp.scala#L368

In spec:
```
trait DependencySourcesItem {
  def target: BuildTargetIdentifier
  /** List of resources containing source files of the
    * target's dependencies.
    * Can be source files, jar files, zip files, or directories. */
  def sources: List[Uri]
}
```

I'm guessing bsp4j is the wrong one?

**Comment by jastice** [comment]
Thanks, I'll just wait on that release then!

**Comment by olafurpg** [comment]
We are still ironing out the interrop between the two libraries and adding test suites for every endpoint. I hope we can have a new release out in the coming week.

**Comment by jvican** [comment]
Fixed in #36. The fix is also already available as v1.1.0 in Maven Central.
